robotium-solo/src/main/java/com/robotium/solo/SystemUtils.java

```diff
@@ -69,19 +69,15 @@ public void setMobileData(Boolean turnedOn){
 	}
 
         /**
-	 * Sets if mobile wifi should be turned on or off. Requires following permissions in the AndroidManifest.xml of the application under test. 
-	 * 1. android.permission.ACCESS_WIFI_STATE 
-	 * 2. android.permission.CHANGE_WIFI_STATE 
+	 * Sets if mobile wifi should be turned on or off. Requires android.permission.CHANGE_WIFI_STATE permissions in the AndroidManifest.xml of the application under test. 
```

Comment:
What about caching Exception? It will interrupt test execution, if there is no permission